### ISSUE TITLE
bug: ios 14.5, variables changes not being rendered with pointer-events property

### ISSUE THREAD

**Issue by zzow** [ionic-team/ionic-framework#23256]
**Ionic version:**

[] **4.x**
[x] **5.x**

**I'm submitting a ...** 

[x] bug report
[ ] feature request

**Current behavior:**
Since the ios update 14.5 the text color of the ion-button do not change when he is disabled.
That's my disabled and normal button:
![image](https://user-images.githubusercontent.com/30020455/116859147-d09ea480-abff-11eb-8805-a0b09a6f6616.png) ![image](https://user-images.githubusercontent.com/30020455/116859447-4f93dd00-ac00-11eb-9fa9-f2bde26820f6.png)

That's the button on my iPhone:
![image](https://user-images.githubusercontent.com/30020455/116859678-aa2d3900-ac00-11eb-8313-de089aa2756b.png)
The button is enabled first and got disabled by setting the disabled attribute to true.
I can reproduce the bug in both directions enabled <> disabled.

**Expected behavior:**
The text color should set to correct value.

**Steps to reproduce:**

1. Add a ion-button to template
2. Bind the disabled attribute to a property
3. Change the disabled property after the component rendered

**Related code:**
@see https://github.com/zzow/ionic-issue-23256

**Ionic info:** 

```
Ionic:

   Ionic CLI                     : 6.13.1
   Ionic Framework               : @ionic/angular 5.6.6
   @angular-devkit/build-angular : 0.901.12
   @angular-devkit/schematics    : 9.1.12
   @angular/cli                  : 9.1.12
   @ionic/angular-toolkit        : 2.3.3

Capacitor:

   Capacitor CLI   : 2.4.4
   @capacitor/core : 2.4.4

Utility:

   cordova-res : 0.15.3
   native-run  : not installed

System:

   NodeJS : v14.16.0
   npm    : 6.14.11
   OS     : Windows 10
```

**Comment by liamdebeasi** [comment]
Thanks for the issue. Can you reproduce this in an Ionic starter app and provide a link to the repo? I am unable to reproduce the issue using the info provided.

**Comment by zzow** [comment]
> Thanks for the issue. Can you reproduce this in an Ionic starter app and provide a link to the repo? I am unable to reproduce the issue using the info provided.

I pushed a project from starter template. https://github.com/zzow/ionic-issue-23256

**Comment by liamdebeasi** [comment]
Thanks! I can reproduce this behavior. This looks to be a bug in WebKit, the engine that powers Safari and WKWebView on iOS. It looks like this regressed with iOS 14.5, as I cannot reproduce the issue on iOS 14.4.

I have reported this issue to the WebKit team: https://bugs.webkit.org/show_bug.cgi?id=225354

**Comment by liamdebeasi** [comment]
This issue appears to be resolved in the iOS 14.7 developer beta. I will post here when iOS 14.7 has shipped.

**Comment by liamdebeasi** [comment]
Hi everyone,

iOS 14.7 has been released, which includes a fix for this bug. You can access this update via Settings > General > Software Update.

For any additional issues in Ionic Framework, please open a new issue. Thank you!
